LibreOfficeWriter_Frame.au3

```diff
@@ -8,23 +8,12 @@
 #include "LibreOfficeWriter_Doc.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Creating, Modifying, and Applying L.O. Writer Frame Styles. Also for Creating, and Modifying Frames.
```

Comment:
Frames ?

Reply:
Yes this file deals with both Frames and Frame Styles.

Reply:
my mistake. Sorry.